python-main.js

```diff
@@ -1156,23 +1156,24 @@ function web_editor(config) {
             } 
         }
 
-        // Show error message
-        $("#flashing-overlay-error").html(
-            '<div>' +
-                '<strong>' + ((err.message === undefined) ? "WebUSB Error" : err.message) + '</strong><br>' + 
-                errorDescription +
-                '<div class="modal-msg-links">' +
+        var errorHTML = '<div>' + ((err.message === undefined) ? "WebUSB Error" : err.message) + '<br >' + 
+                    errorDescription + '<div class="flashing-overlay-buttons">' +
                     (err.name === 'device-disconnected'
                             ?  ""
-                            : '<br ><a href="#" id="flashing-overlay-download">' +
-                            config["translate"]["webusb"]["download"] +
-                            '</a> | ') +
+                            : '<a href="#" id="flashing-overlay-download">' +
+                              config["translate"]["webusb"]["download"] + 
+                              '</a> | ') +
                     '<a href="#" onclick="flashErrorClose()">' +
                     config["translate"]["webusb"]["close"] +
-                    '</a>' +
-                '</div>' +
-            '</div>'
-        );
+                    '</a></div></div>';
+
+        // Show error message
+        if($("#flashing-overlay-error").html() == "") {
+            $("#flashing-overlay-error").html(errorHTML);
+        } else {
+            $(".flashing-overlay-buttons").hide(); // Hide previous buttons
+            $("#flashing-overlay-error").append("<hr />" + errorHTML);
```

Comment:
Should we add another hr between the text and the buttons?
Otherwise it feels like the buttons are only relevant to the last message.

Reply:
Added this